tests/read/coff.rs

```diff
@@ -21,3 +21,20 @@ fn coff_extended_relocations() {
     let relocations = code_section.relocations().collect::<Vec<_>>();
     assert_eq!(relocations.len(), 65536);
 }
+
+
+#[cfg(feature = "coff")]
+#[test]
+fn coff_symbol_raw_test() {
```

Comment:
Thanks for the diligence with regard to testing. However, I prefer not to have tests for trivial getter functions, so let's leave this out.